CDTDatastore.xcodeproj/project.pbxproj

```diff
@@ -10,6 +10,14 @@
 		016D1893CDC0214E9D985B6A /* Pods_CDTDatastoreEncryptedReplicationAcceptanceTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 1E1355F12F8B3C6BAD03877E /* Pods_CDTDatastoreEncryptedReplicationAcceptanceTests.framework */; };
 		093B0D76D4A2A4C1B1573A14 /* Pods_CDTDatastoreEncryption.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 1E7A2ACFEFC64F349EC3C349 /* Pods_CDTDatastoreEncryption.framework */; };
 		2D4884327E9CA7EC3E066969 /* Pods_CDTDatastoreEncryptionOSX.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 486CE35D1A1E051980A59D58 /* Pods_CDTDatastoreEncryptionOSX.framework */; };
+		3567D22135DB02790BD939BA /* CDTDatastore+Replication.h in Headers */ = {isa = PBXBuildFile; fileRef = 3567D9F9C835096137DC8EF2 /* CDTDatastore+Replication.h */; };
```

Comment:
This should be a public header so it's exported when CDTDatastore is built as a Framework